setup.py

```diff
@@ -98,6 +98,8 @@ def _get_readme():
                 'pycodestyle>=2.3.1',
                 'pylint>=1.6.0,<2.0',
                 'requests_mock>=1.0.0,<2.0',
+                'coverage==4.5.1',
```

Comment:
Why such a specific version pin

Reply:
I picked the latest version atm. 

https://coverage.readthedocs.io/en/coverage-4.5.1/changes.html

Reply:
So if we had picked up this work next month instead of today then the pin would be different.... That's a bit too serendipitous for me

Reply:
True. I have the same feeling for `pycodestyle`.  Maybe I should use `coverage>=4.5.1` 

But I don't know whether we should keep pursuing the latest version of a package. Since the latest version is not really guaranteed bug-free and it might have conflicts with other packages.

Reply:
I agree pinning this to a specific version is not a good idea. See the comment above this in setup.py for general policy; let's do `>=4.5.1,<5.0.0`.